docs/reference/templates.rst

```diff
@@ -329,8 +329,8 @@ contains all the specifications needed by the ``renku init`` function to
 create a new project. You can specify multiple templates in the same
 repository. Each of them requires an entry with the following parameters:
 
-* ``folder``: the target folder inside the repository where the template files
-  are stored. Please use a different folder for each template.
+* ``id`` (instead of ``folder`` which is deprecated): the target directory inside the repository
```

Comment:
What happens to template repositories in the "wild" where they still use `folder`? Is the grace period for switching already done and from now on you have to use `id` and only using `folder` will fail?

Reply:
No. `folder` will be supported forever. We only print a warning to replace it with `id` when validating a template using `renku template validate` command.